### ISSUE TITLE
Add support for alternative pool calculations, like some Lunar Charms allow

### ISSUE THREAD

**Issue by makzu** [makzu/lotcastingatemi#41]

**Comment by bugQ** [comment]
Yes, this would be nice. Documenting all such permanent charms that I can find here:

| charm | pool | replace | with | condition |
| --- | --- | --- | --- | --- |
| Deadly Claw Blow | Withering Attack, Decisive Attack | Dexterity | Strength | non-ranged, unarmed or heavy or natural weapon |
| Angry Rhino Charge | Rush, Disengage | Dexterity | Strength | |
| Steel Paw Style | Parry | Dexterity | Stamina | unarmed or medium or heavy or natural weapon |
| Arrogant Lion Bearing | Resolve | Wits | Charisma | |
| Unwavering Predator's Eye | Join Battle | Wits | Perception | |
| Sublime Façade Art | Guile | Socialize | Integrity or Larceny or Performance | |
